css.webpack.js

```diff
@@ -57,48 +35,29 @@ module.exports = (env) => {
           ]
         },
         {
+          // this inlines resources if they are less than 8KB
           test: /\.(png|jpe?g|gif|svg|eot|ttf|woff|woff2)$/i,
-          loader: 'url-loader',
-          options: {
-            limit: 8192,
-          },
+          type: 'asset'
         }
       ]
     },
     plugins: [
       // extracts CSS to a separate file
       new MiniCssExtractPlugin({
-        filename: env.development ? '../stylesheets/[name].css' : '../stylesheets/[name].[hash].css',
+        filename: env.development ? '../stylesheets/[name].css' : '../stylesheets/[name].[contenthash].css',
       }),
       // generates a RTL version of the emitted CSS files
-      // this is done only in production
-      ...(env.production || env.coverage ? [new WebpackRTLPlugin({
-        filename: '../stylesheets/rtl-[name].[contenthash].css'
+      // this is done only in production/coverage
+      ...(env.production || env.coverage ? [new RtlCssPlugin({
+        filename: '../stylesheets/rtl-[name].[fullhash].css'
       })] : []),
-      // css-loader generates unnecesary js files (but with .styl extension)
-      // the following will remove that
-      new ExcludeAssetsPlugin({
-        path: ['^.*\\.styl$']
-      }),
-      // manifest file
-      new ManifestPlugin({
-        fileName: 'css-manifest.json',
-        map: (file) => {
-          if (/rtl-.*\.css$/.test(file.path)) {
-            file.name = `rtl-${file.name}`;
-          }
-          return file;
-        }
-      }),
-      // node environment
-      new webpack.DefinePlugin({
-        'process.env': {
-          NODE_ENV: JSON.stringify(mode),
```

Comment:
[It is](https://webpack.js.org/plugins/define-plugin/) but it looks like since Webpack 4 defining `mode` should set `process.env.NODE_ENV` [automatically](https://webpack.js.org/guides/production/#specify-the-mode) - so this might've been redundant as well.